autoload/sj/php.vim

```diff
@@ -187,8 +187,24 @@ function! s:JoinList(start_char, end_char)
   return 1
 endfunction
 
+function! s:SplitNextArrow()
+  let l:arrow_or_paren = search('\v(\(|\S\zs-\>\ze)', '', line('.'))
+
+  if ! arrow_or_paren
+    return
+  endif
+
+  if matchstr(getline('.'), '\%' . col('.') . 'c.') == '('
+    normal! %
+  else
+    normal! i

```

Comment:
You can replace this with:

``` vim
exe "normal! i\<cr>"
```

It's the same thing, but the special character shows up weird in diffs and such, so I'd rather use a string-escaped `\<cr>`